manifest.md

```diff
@@ -132,14 +132,14 @@ A client will distinguish a manifest list from an image manifest based on the Co
 
 # Image Manifest
 
-The image manifest provides a configuration and a set of layers for a container image.
+Unlike the [(Manifest List](#manifest-list), which contains information about a set of images that can span a variety of archiectures and operating systems, an image manifest provides a configuration and set of layers for a single container image for a specific architecture and operating system.
```

Comment:
fixed!

> On May 6, 2016, at 11:31 AM, Brandon Philips <notifications@github.com> wrote:
> 
> In manifest.md <https://github.com/opencontainers/image-spec/pull/68#discussion_r62346141>:
> 
> > @@ -132,14 +132,14 @@ A client will distinguish a manifest list from an image manifest based on the Co
> >  
> >  # Image Manifest
> >  
> > -The image manifest provides a configuration and a set of layers for a container image.
> > +Unlike the [(Manifest List](#manifest-list), which contains information about a set of images that can span a variety of archiectures and operating systems, an image manifest provides a configuration and set of layers for a single container image for a specific architecture and operating system.
> nit: spelling error: architectures
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/opencontainers/image-spec/pull/68/files/9494cb085a34ec3e10a2ab5ccdf528184585c0b4#r62346141>